core/types/transaction.go

```diff
@@ -49,24 +48,28 @@ type txdata struct {
 	Recipient       *common.Address `rlp:"nil"` // nil means contract creation
 	Amount          *big.Int
 	Payload         []byte
-	V               byte     // signature
-	R, S            *big.Int // signature
+	V, R, S         *big.Int // signature
```

Comment:
I guess because it's used in operations with other bit.Ints, so should be converted at some point before that